src/implementations/twig/components/select/select.html.twig

```diff
@@ -58,10 +58,10 @@
 {% set _css_group_class = 'ecl-form-group' %}
 {% set _css_class = ['ecl-select'] %}
 {% set _css_class = _css_class|join(' ') %}
-{% set _extra_attributes = '' %}
-{% set extra_attributes = [{ name: 'data-ecl-auto-init', value: 'Select' }] %}
+{% set extra_attributes = extra_attributes|default([]) %}
+{% set extra_attributes = extra_attributes|merge([{ name: 'data-ecl-auto-init', value: 'Select' }]) %}
```

Comment:
We can define it only once as:
` {% set extra_attributes = extra_attributes|default([])|merge([{ name: 'data-ecl-auto-init', value: 'Select' }]) %}`